pom.xml

```diff
@@ -168,7 +168,7 @@
                         <![CDATA[<a href='https://drewnoakes.com/code/exif/' title='Go to the project home page.'><img src='https://raw.githubusercontent.com/drewnoakes/metadata-extractor/master/Resources/metadata-extractor-logo-131x30.png' border="0" alt='Metadata Extractor Logo'></a>]]></header>
                     <bottom><![CDATA[<i>Copyright &#169; 2002-2017 Drew Noakes. All Rights Reserved.</i>]]></bottom>
                     <excludePackageNames>com.drew.tools</excludePackageNames>
-                    <stylesheetfile>Resources/javadoc-stylesheet.css</stylesheetfile>
+                    <stylesheetfile>${project.basedir}/Resources/javadoc-stylesheet.css</stylesheetfile>
```

Comment:
Looks good. Was there a problem that this fixed for you?

Reply:
Yes, I used metadata-extractor as a module in a pom-packaging (parent/multi-child):
	<packaging>pom</packaging>
	<modules>
		<module>metadata-extractor</module>
		<module>xyz</module>
	</modules>
This change fixed the path for the stylesheetfile.

Reply:
Nice, thanks.